riscv/src/continuations.rs

```diff
@@ -435,27 +455,26 @@ pub fn rust_continuations_dry_run<F: FieldElement>(
             (length - start - shutdown_routine_rows) * 100 / length
         );
         for i in 0..(chunk_trace["main.pc"].len() - start) {
-            for &reg in REGISTER_NAMES.iter() {
-                let chunk_i = i + start;
-                let full_i = i + proven_trace;
-                if chunk_trace[reg][chunk_i] != full_trace[reg][full_i] {
-                    log::error!("The Chunk trace differs from the full trace!");
-                    log::error!(
+            let &reg = &"main.pc";
```

Comment:
Hmm, so we used to compare all register, now only the PC.

Can we instead loop over all columns keys of `chunk_trace`? I think it would also contain extra registers, right?

Reply:
Yea true, we're not handling the extra registers properly. I'd prefer to do the continuations stuff in a separate PR.

Reply:
Not sure what you mean, if we did the check on all columns of `chunk_trace`, would it fail? I think not?

Reply:
added val1 and val2